Makefile

```diff
@@ -509,6 +510,14 @@ go-postgres-unit-tests: build-prep test-prep
 		$(shell git grep -rl "//go:build sql_integration" central pkg migrator tools | sed -e 's@^@./@g' | xargs -n 1 dirname | sort | uniq | xargs go list -tags sql_integration | grep -v '^github.com/stackrox/rox/tests$$' | grep -Ev $(UNIT_TEST_IGNORE)) \
 		| tee $(GO_TEST_OUTPUT_PATH)
 
+.PHONY: go-postgres-bench-tests
+go-postgres-bench-tests: build-prep test-prep
+	@# The -p 1 passed to go test is required to ensure that tests of different packages are not run in parallel, so as to avoid conflicts when interacting with the DB.
+	set -o pipefail ; \
+	CGO_ENABLED=1 GODEBUG=cgocheck=2 MUTEX_WATCHDOG_TIMEOUT_SECS=30 ROX_POSTGRES_DATASTORE=true GOTAGS=$(GOTAGS),test,sql_integration scripts/go-test.sh -p 1 -run=nonthing -bench=. -benchtime=$(BENCHTIME) -benchmem -timeout 20m -v \
```

Comment:
nitpick: It would be good to pull out timeout (`20m`) to an env variable so that we have the flexibility to set it without changing Makefile.

Reply:
good point allow configuring it and added a readme with info about it